## REQUEST
# This service is used to retrive the infomation
# regarding a room using an aruco marker id.

# The aruco marker id.
int32 id

---

## RESPONSE

# The room the aruco marker was related to.
string room

# The coordinates of the room.
float32 x
float32 y

# The connections the room has.
RoomConnection[] connections
